assets/css/style.css

```diff
@@ -1,23 +1,25 @@
-main {
-  background-image: url(/assets/images/header-shapes-mobile.png);
+body {
+  margin: 0;
+  padding: 0;
+  max-width: 375px;
+  background-color: #f7f7f9;
 }
```

Comment:
- [ ] Kindly remove the **max-width** to make your project expandable to the whole screen. I know responsiveness is not a part of this project but your project should be responsive on all mobile screens :blush:

Reply:
It's Done!